tests/ui_tests/app/configuration.py

```diff
@@ -55,16 +68,40 @@ def assert_field_is_editable(self, field_element, editable=True) -> None:
             self.is_element_editable(field_element) == editable
         ), f"Field {field_element} should {'be editable' if editable else 'not be editable'}"
 
+    def get_api_value(self, adcm_credentials, app_fs, field: str):
+        """Gets field value by api"""
+
+        current_config = _make_request(
```

Comment:
Why do you want to get it directly via API?

Reply:
how do you suggest?

Reply:
I think fields that you're using may be found via the client's `.confg(full=True)` call. I suggest checking it to reduce functionality duplication in code

Reply:
removed this func